client_cli/pytests/common/helpers.py

```diff
@@ -0,0 +1,56 @@
+import binascii
+import random
+import string
+
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric.ed25519 import Ed25519PrivateKey
+from common.consts import Name, fake
+
+
+
+def generate_random_string_with_reserved_char():
+    string = fake.word()
+    letter = random.choice(Name.RESERVED_CHARS.value)
+    random_position = random.randint(0, len(string))
+    new_string = string[:random_position] + letter + string[random_position:]
+    return new_string
+
+def generate_random_string_with_whitespace():
+    string = fake.word()
+    letter = random.choice(Name.WHITESPACES.value)
+    random_position = random.randint(0, len(string))
+    new_string = string[:random_position] + letter + string[random_position:]
+    return new_string
+
+def generate_public_key():
+    public_key = binascii.hexlify(
+        Ed25519PrivateKey.generate().public_key().public_bytes(
+            encoding=serialization.Encoding.Raw,
+            format=serialization.PublicFormat.Raw)).decode()
+    return public_key
```

Comment:
could/should we utilize `kagami` for this purpose?

Reply:
i think so, we have `kagami` json mode

Reply:
Actually, using Python native libs is good.